### DISCUSSION TITLE
Question: GeyserMC Player Register?

### DISCUSSION THREAD

**Discussion by pixlflip** [TownyAdvanced/Towny#4568] [Q&A] [answered]
Hello. I run a mid sized server which allows cross play between bedrock and java using the GeyserMC plugin for Spigot. My question is this: when I added Towny to the server, all bedrock players come up as “not registered” and cannot join, create, or use towns. Is there something I can do to fix this, and if not, could this be fixed in a future patch to Towny? Thanks!

**Comment by LlmDl** [comment, marked as answer]
Geyser can be used without floodgate. Geyser is not ready for production servers. You can disregard this message if you can figure out how to change the towny regex to allow * in the names (still not recommended.) I am not paid enough to deal with floodgate servers.

**Reply by pixlflip** [reply]
Thank you for your help!
